assets/src/models/routeTab.ts

```diff
@@ -3,7 +3,27 @@ import { LadderDirections } from "./ladderDirection"
 import { LadderCrowdingToggles } from "./ladderCrowdingToggle"
 
 export interface RouteTab {
+  isCurrentTab: boolean
+  presetName?: string
+  id?: string
   selectedRouteIds: RouteId[]
   ladderDirections: LadderDirections
   ladderCrowdingToggles: LadderCrowdingToggles
 }
+
+export const newRouteTab = (): RouteTab => ({
+  isCurrentTab: true,
+  selectedRouteIds: [],
+  ladderDirections: {},
+  ladderCrowdingToggles: {},
+})
+
+export const currentRouteTab = (routeTabs: RouteTab[]): RouteTab => {
+  const currentTab = routeTabs.find((routeTab) => routeTab.isCurrentTab)
```

Comment:
Good catch. FWIW I'm probably going to have to rework this logic at some point anyway, because simply calling `newRouteTab()` doesn't actually add it to the state, and I need to figure out a graceful spot to put that.